Rakefile

```diff
@@ -0,0 +1,53 @@
+#
+# Author:: John McCrae (<john.mccrae@progress.com>)
+# Copyright:: Copyright, Chef Software Inc.
+# Copyright:: Copyright, Progress Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+begin
+  require "rubocop/rake_task"
+  require "rspec/core/rake_task"
+  require "rubygems"
+  require "rake"
+  require "chefstyle"
+rescue LoadError => e
+  puts "Skipping missing rake dep: #{e}"
+end
+
+desc "Builds and Copies powershell_exec related binaries from the latest built Habitat Packages"
+task :update_chef_powershell_dlls do
+  raise "This task must be run on Windows since we are installing a Windows targeted package!" unless Gem.win_platform?
+
+  require "mkmf"
+  raise "Unable to locate Habitat cli. Please install Habitat cli before invoking this task!" unless find_executable "hab"
+
+  sh("hab pkg build Habitat")
+  sh("hab pkg build Habitat-x86")
+
+  sh("hab pkg install chef/chef-powershell-shim")
+  sh("hab pkg install chef/chef-powershell-shim-x86")
+
```

Comment:
you build it locally and then install the latest stable version which I don't think is what you want to do. Alsodo we need this task since the expeditor script is doing the same thing?

Reply:
I added that part under the assumption that we might want to trigger a build if only either the DLL's or the gem files were changed but it makes more sense to remove that. Now removed